utils/namespaces.js

```diff
@@ -0,0 +1,33 @@
+import { sortBy } from '@/utils/sort';
+
+// This method is used by the NameNsDescription component and by the create/edit
+// workload form to provide namespaces to choose from.
+const getAvailableNamespaces = (resource) => {
+  const inStore = resource.$store.getters['currentStore'](resource.namespaceType || 'namespace');
+  const choices = resource.$store.getters[`${ inStore }/all`](resource.namespaceType || 'namespace');
+
+  const out = sortBy(
+    choices.filter( (N) => {
```

Comment:
By giving a meaningful and extended name, i.e. `choice` or `namespace` following the previous suggestion, it may be easier to read.